### DISCUSSION TITLE
the age limit

### DISCUSSION THREAD

**Discussion by hubhubb** [mozilla/hubs#5280] [General Questions & Discussion] [answered]
I wanna know for sure the age available to students.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1225)

**Comment by camelman90** [comment, marked as answer]
This depends entirely on what laws for electronic communication, and related laws, guidelines and policy is relevant in your area / country and also the country / area where the data is being processed and where the end users are located. It is also dependent on what features (voice, video, private / public room, etc) and content you chose to apply in your room, as well as the consent gathered from participants or other authority or responsible (under teacher or parental guidance, with parental permission). There is no simple answer for this because it is relative to your area and your laws. 

I suggest you consult your government data protection/authority offices or a lawyer.
